frontend/src/pages/RunDetails.tsx

```diff
@@ -374,6 +390,7 @@ class RunDetails extends Page<RunDetailsProps, RunDetailsState> {
         buttons.restore(idGetter, true, () => this.refresh()) :
         buttons.archive(idGetter, true, () => this.refresh());
       actions.splice(2, 1, newButton);
```

Comment:
Is this index still correct now that we have one more button before Archive/Restore?

Reply:
Good call. No, it was removing the refresh button.

Do we need a refresh button though now that we have autorefresh? it would seem not